integration/examples/helm-remote-repo/README.md

```diff
@@ -0,0 +1,20 @@
+### Example: deploy remote helm chart
+
+This example shows how to deploy a remote helm chart from a remote repo. This can be helpfulf or consuming other helm packages as part of your app.
```

Comment:
```suggestion
This example shows how to deploy a remote helm chart from a remote repo. This can be helpful  for consuming other helm packages as part of your app.
```